tap_stripe/__init__.py

```diff
@@ -705,6 +705,27 @@ def sync_sub_stream(sub_stream_name, parent_obj, updates=False):
             obj_ad_dict = sub_stream_obj.to_dict_recursive()
 
             if sub_stream_name == "invoice_line_items":
+                # we will get "unique_id" for default API versions older than "2019-12-03"
+                if updates and obj_ad_dict.get("unique_id"):
+                    # get unique_id
+                    unique_id = obj_ad_dict.get("unique_id")
+                    # if type is invoiceitem, update 'id' field with 'unique_id'
+                    if obj_ad_dict.get("type") == "invoiceitem":
+                        # get invoice_item id
+                        invoice_item_id = obj_ad_dict.get("id")
+                        obj_ad_dict["id"] = unique_id
+                        # update 'invoice_item' with 'id' if not present
+                        if not obj_ad_dict.get("invoice_item"):
+                            obj_ad_dict["invoice_item"] = invoice_item_id
+                    # if type is subscription, update 'id' field with 'unique_id'
+                    if obj_ad_dict.get("type") == "subscription":
```

Comment:
Can we use elif here? If both conditions will never get true together.

Reply:
Updated.